pytest_hoverfly/base.py

```diff
@@ -17,6 +17,7 @@
 
 IMAGE = "spectolabs/hoverfly:v1.3.2"
 CONTAINER_BASENAME = "test-hoverfly"
+SERVICE_HOST_VARIABLE_NAME = "SERVICE_HOST"
```

Comment:
I would honestly just hardcode it. It's not like we're going to reuse this anywhere.

Reply:
Ok